pkg/agent/controller/egress/ipassigner/ip_assigner_linux.go

```diff
@@ -117,7 +128,7 @@ func (a *ipAssigner) AssignIP(ip string) error {
 			return nil
 		}
 
-		addr := netlink.Addr{IPNet: &net.IPNet{IP: parsedIP, Mask: net.CIDRMask(32, 32)}}
+		addr := netlink.Addr{IPNet: &net.IPNet{IP: parsedIP, Mask: net.CIDRMask(maskLen, maskLen)}}
```

Comment:
You can use `netlink.NewIPNet(parsedIP)`

Reply:
Got it, done